styles/base.css

```diff
@@ -1,18 +1,22 @@
 @import url("./fonts.css");
 
 :root {
-    --green: #88B440;
     --white: #FFFFFF;
-    --brand-primary-cyan: #51B1C7;
+    --text-black: #1C1C1C;
+    --brand-primary-green: #88B440;
     --brand-primary-black: #1C1C1C;
-    --bs-light-rgb: 84, 84, 84; /* for the HR above Footer */
-    --bs-light: #545454; /* brand/scales/gray/4 */
+    --brand-primary-cyan: #51B1C7;
     --bs-primary: #88B440; /* Use with .text-primary */
     --bs-primary-rgb: 136, 180, 64;
     --bs-secondary: #51B1C7; /* Use with .text-secondary */
     --bs-secondary-rgb: 81, 177, 199;
+    --bs-light: #545454; /* Use with .border-light above Footer - /brand/scales/gray/4 */
+    --bs-light-rgb: 84, 84, 84;
+    --bs-info: #BEB3B3; /* Use with .text-info */
+    --bs-info-rgb: 190,190,190;
     --bs-font-sans-serif: "Roboto", system-ui,-apple-system,"Segoe UI","Helvetica Neue","Noto Sans","Liberation Sans",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol","Noto Color Emoji";
     --bs-font-monospace: "Source Code Pro Bold",monospace;
+    --focus-box-shadow: 0 0 0 calc(2.5rem / 16) var(--brand-primary-black), 0 0 0 calc(4rem / 16) var(--brand-primary-green);
```

Comment:
This code creates this effect as a box-shadow:
<img width="730" alt="image" src="https://github.com/compilerla/compiler.la/assets/3673236/7c62cf38-c77f-4527-a6ec-303d44c48e7d">


How to read this code:
- `0 0 0 calc(2.5rem / 16) var(--brand-primary-black)`: Create a 2.5px-thick black line, the same color as the background color.
-  `0 0 0 calc(4rem / 16) var(--brand-primary-green)`: _Below_ that first line, _starting from the same point_, add another 4px of green line. Only 1.5px of green will actually show up. That's calculated like this: 4-2.5 = 1.5.